### PR TITLE
Update qdk-relnotes.md

### PR DESCRIPTION
**Pull request by KittyYeungQ**

### CODE REVIEW

File: articles/qdk-relnotes.md [hunk 1]
```diff
@@ -19,8 +19,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
+
 ## Version 0.15.2103.133969
 
+*Release date: April 22nd, 2021*
+
+- HM type inference algorithm.
+- Added support for empty array literals.
```
Comment (samarsha) [inline articles/qdk-relnotes.md:28]:
```suggestion
- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
```

File: articles/qdk-relnotes.md [hunk 2]
```diff
@@ -19,8 +19,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
+
 ## Version 0.15.2103.133969
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:23]:
```suggestion
## Version 0.16.xxxx.xxxxxx
```

File: articles/qdk-relnotes.md [hunk 3]
```diff
@@ -19,8 +19,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
 
-## Version 0.15.2103.133969
 
+## Version 0.16.xxxx.xxxxxx
+
+*Release date: April 22nd, 2021*
+
+- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
+- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
+- Added support for more types for 'w' and 'indices' inputs for QIO Python SDK.
```
Comment (ricardo-espinoza) [inline articles/qdk-relnotes.md:29]: I would rephrase this as:
"Added support for NumPy types in coefficient definitions for problems in QIO Python SDK."

File: articles/qdk-relnotes.md [hunk 4]
```diff
@@ -19,8 +19,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Updated control-plane CLI with new swagger to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```
Comment (ricardo-espinoza) [inline articles/qdk-relnotes.md]: Also, recommend changing this to "Updated control-plane swagger file to [support ...](...)"
Let me prepare a separate PR for the CLI items to include past ones as well.

File: articles/qdk-relnotes.md [hunk 5]
```diff
@@ -19,8 +19,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+## Version 0.16.xxxx.xxxxxx
+
+*Release date: April 22nd, 2021*
+
+- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
+- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
+
+- Added support for NumPy types in coefficient definitions for problems in QIO Python SDK.
+
+- Released v0.X.0 of the Az CLI extension: Updated control-plane CLI with new swagger to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:32]:
```suggestion
- Released v0.X.0 of the Az CLI extension: Updated control-plane swagger file to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```

File: articles/qdk-relnotes.md [hunk 6]
```diff
@@ -19,8 +19,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
 
-## Version 0.15.2103.133969
 
+## Version 0.16.xxxx.xxxxxx
+
+*Release date: April 22nd, 2021*
+
+- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
+- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
+
```
Comment (ricardo-espinoza) [inline articles/qdk-relnotes.md:29]: NIT: No need to separate in sections with the blank lines, we can keep everything together.

File: articles/qdk-relnotes.md [hunk 7]
```diff
@@ -19,8 +19,19 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Added support for NumPy types in coefficient definitions for problems in QIO Python SDK.
+
+- Released v0.X.0 of the Az CLI extension: Updated control-plane swagger file to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```
Comment (ricardo-espinoza) [inline articles/qdk-relnotes.md:32]: I'll send a PR into your PR addressing the CLI updates, we can discuss there.

Comment (ricardo-espinoza) [inline articles/qdk-relnotes.md:32]:
```suggestion
- Updated control-plane swagger file to [support restricted access SKUs](https://github.com/Azure/azure-rest-api-specs/pull/13002).
```

File: articles/qdk-relnotes.md (hunk 6 above)
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:29]:
```suggestion
```

File: articles/qdk-relnotes.md [hunk 8]
```diff
@@ -19,8 +19,18 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
-## Version 0.15.2103.133969
 
+## Version 0.16.xxxx.xxxxxx
+
+*Release date: April 22nd, 2021*
+
+- Improved Q# type inference based on the Hindley-Milner type inference algorithm.
+- [QEP 2: Enhanced Array Literals](https://github.com/microsoft/qsharp-language/blob/main/Approved/2-enhanced-array-literals.md) has been implemented.
+- Added support for NumPy types in coefficient definitions for problems in QIO Python SDK.
+
```
Comment (KittyYeungQ) [inline articles/qdk-relnotes.md:30]:
```suggestion
```

**Review by ricardo-espinoza — APPROVED** [review]
